_woc/Welcome to my WoC blog.md

```diff
@@ -0,0 +1,61 @@
+---
+title: "Text Sentiment Analysis Starter Blog"
+excerpt: "This is the inital blog post to derive a generalized interest for the project of Text Sentiment Analysis."
+toc: true
+toc_sticky: true
+toc_label: "Contents"
+
+categories:
+  - woc
+
+tags:
+  - Winter of Code
+  - Developers
+  - Developer Student Clubs
+  - DSC NSEC
+  - DSC IEM
+
+
+og_image: /assets/images/WoC2020logo.jpg
+header:
+  teaser: "/assets/images/WoC2020logo.jpg"
+  overlay_image: /assets/images/WoC2020logo.jpg
+  overlay_filter: 0.5 # same as adding an opacity of 0.5 to a black background
+  caption: "Image credit: [**Winter of Code**](https://winterofcode.com/) on [**Instagram**](https://www.instagram.com/winterofcode/)"
+  actions:
+    - label: "View Repository on Github"
+      url: "https://github.com/khanfarhan10/TextSentimentAnalysis"
+author: Ankita_Sareen
```

Comment:
Just adding the author wont work, you have to change authors.yml in the _data section!
https://github.com/khanfarhan10/khanfarhan10.github.io/blob/master/_data/authors.yml

Or watch the YT video : https://youtu.be/F5_tRRIyyiY